sites/github.styl

```diff
@@ -269,14 +272,37 @@ a.filter-item
 .branch-action-state-dirty
 .branch-action-state-unstable
 .branch-action-state-unknown
-    .branch-action-icon
+.branch-action-icon
         background-color yellow i
         color white i
 
+.text-shadow-light
+.branch-group-name
+.branch-summary
+    color color-text i
```

Comment:
The `color` mixin applies `color-text i` automatically, so you can change this to just `color`.